src/minimega/qos_cli.go

```diff
@@ -40,8 +40,8 @@ Examples:
 		Patterns: []string{
 			"qos <add,> <target> <interface> <loss,> <percent>",
 			"qos <add,> <target> <interface> <delay,> <duration>",
-			"qos <add,> <target> <interface> <rate,> <bw>",
-		}, Call: wrapVMTargetCLI(cliQos),
+			"qos <add,> <target> <interface> <rate,> <kbit, mbit, gbit>",
```

Comment:
Probably shouldn't have spaces between units -- I *think* minicli will trim them but I'm not certain.